### PR TITLE
added grailpay icon

### PR DESCRIPTION
**Pull request by AnoopChauhan97**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [x] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by messinagp** [comment]
@larouxn Our PR is updated and ready for review, let me know what you think!

**Comment by AnoopChauhan97** [comment]
@messinagp Our PR is ready for review. So how much time does it gonna take?

**Review by larouxn — CHANGES_REQUESTED** [review]
The border is a bit off. I think it might because it's an outward facing border or something. I'm not sure of the specifics. 
Screen Shot 2022-06-01 at 13 49 00

Please refer to the [template](https://github.com/activemerchant/payment_icons/blob/3b558b771e1e096b8c4c23f0fef13f195563a19d/CONTRIBUTING.md#template).

**Comment by messinagp** [comment]
Hi @larouxn we updated as requested. Let me know if all looks set! Thanks

**Comment by messinagp** [comment]
Hey 👋 @larouxn we should be all good - thank you!

**Comment by messinagp** [comment]
Hey @larouxn I see you’re merging some PRs. It would be very helpful if GrailPay could be merged in this batch as well. Thanks so much

**Review by larouxn — APPROVED** [review]
Looks good to me 🎩 
Screen Shot 2022-06-09 at 17 06 20

File: app/assets/images/payment_icons/grailpay.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...31133364857491_" gradientUnits="userSpaceOnUse" x1="21.524" y1="15.276" x2="32.633" y2="10.399"><stop offset="0" stop-color="#00a035"/><stop offset="1" stop-color="#00d145"/></linearGradient><path d="M28.65 11.002h-.626a.187.187 0 00-.188.186v.31c-.09-.17-.43-.563-1.05-.563s-1.455.44-1.455 1.707c0 1.268.852 1.721 1.455 1.721s.96-.395 1.05-.57v.31c0 .103.084.186.188.186h.626a.187.187 0 00.188-.185v-2.916a.187.187 0 00-.188-.186zm-1.559 2.605c-.648 0-.731-.634-.731-.965 0-.326.082-.95.731-.95s.738.624.738.95c0 .33-.09.965-.738.965z" fill="url(#pi-SVGID_00000043436590437117869210000010855931133364857491_)"/><linearGradient id="pi-SVGID_00000175291399114651225350000013803905921597476495_" gradientUnits="userSpaceOnUse" x1="22.194" y1="16.8" x2="33.303" y2="11.923"><stop offset="0" stop-color="#00a035"/><stop offset="1" stop-color="#00d145"/></linearGradient><path d="M32.812 11.002h-.6a.188.188 0 00-.175.115l-.812 1.974-.786-1.972a.188.188 0 00-.175-.117h-.662a.186.186 0 00-.172.26l1.253 2.847-.516 1.123a.186.186 0 00.172.263h.628a.188.188 0 00.172-.11l1.845-4.122a.186.186 0 00-.172-.261z" fill="url(#pi-SVGID_00000175291399114651225350000013803905921597476495_)"/></g></g></switch></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/grailpay.svg:1]: Could we remove the `<g>` and `<switch>`? These are mostly generated by the editing tool.

```suggestion
<svg width="38" height="24" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 24" role="img" xml:space="preserve" aria-labelledby="pi-grailpay"><title id="pi-grailpay">GrailPay</title><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32m0-1H3C1.3 0 0 1.3 0 3v18c0 1.7 1.3 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.3-3-3-3z" opacity=".07"/><path d="M35 23H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2z" fill="#fff"/><path d="M20.382 13.533c-.048 0-.627.13-.627-.53V9.756a.187.187 0 00-.188-.186h-.646a.187.187 0 00-.188.186v3.29c0 1.318 1.029 1.377 1.625 1.28a.184.184 0 00.151-.18s.048-.612-.127-.612zM9.051 11.685h-1.52a.187.187 0 00-.188.186v.378c0 .102.084.186.188.186h.841v.924c-1.707.561-2.307-.58-2.275-1.413.09-2.377 2.456-1.337 2.604-1.249.1.06.23.02.274-.086l.195-.467a.184.184 0 00-.066-.224c-.474-.38-1.917-.657-2.932-.124S5 11.446 5 11.946c0 .486.212 1.62 1.141 2.116 1.148.612 2.781.082 2.99-.01a.184.184 0 00.108-.169v-2.012a.187.187 0 00-.188-.186zm3.033-.758s-.851 0-1.126.644v-.383a.187.187 0 00-.188-.186h-.626a.187.187 0 00-.188.186v2.916c0 .102.084.186.188.186h.666a.187.187 0 00.188-.186v-1.515c-.021-.898 1.154-.805 1.154-.805.1-.01.175-.098.168-.198l-.035-.486a.187.187 0 00-.2-.173zm3.884.075h-.626a.187.187 0 00-.188.186v.31c-.09-.17-.386-.563-1.05-.563s-1.456.477-1.456 1.707.821 1.721 1.456 1.721.96-.395 1.05-.569v.31c0 .102.084.186.188.186h.626a.187.187 0 00.188-.186v-2.916a.187.187 0 00-.188-.186zm-1.56 2.605c-.626 0-.73-.634-.73-.965 0-.836.514-.95.73-.95.663 0 .739.624.739.95 0 .33-.076.965-.738.965zm3.323-2.605h-.646a.187.187 0 00-.188.186v2.916c0 .102.084.186.188.186h.646a.187.187 0 00.188-.186v-2.916a.187.187 0 00-.188-.186z" fill="#041427"/><ellipse cx="17.408" cy="10.144" rx=".581" ry=".575" fill="#041427"/><linearGradient id="pi-SVGID_1_" gradientUnits="userSpaceOnUse" x1="20.332" y1="12.559" x2="31.441" y2="7.682"><stop offset="0" stop-color="#00a035"/><stop offset="1" stop-color="#00d145"/></linearGradient><path d="M23.422 9.569H21.43a.187.187 0 00-.188.186v4.349c0 .102.084.185.188.185h.673a.187.187 0 00.188-.185v-1.515h1.13c.534 0 1.667-.226 1.667-1.514S23.955 9.57 23.422 9.57zm-.176 2.216h-.955v-1.406h.955c.555 0 .833.232.833.696 0 .473-.278.71-.833.71z" fill="url(#pi-SVGID_1_)"/><linearGradient id="pi-SVGID_00000043436590437117869210000010855931133364857491_" gradientUnits="userSpaceOnUse" x1="21.524" y1="15.276" x2="32.633" y2="10.399"><stop offset="0" stop-color="#00a035"/><stop offset="1" stop-color="#00d145"/></linearGradient><path d="M28.65 11.002h-.626a.187.187 0 00-.188.186v.31c-.09-.17-.43-.563-1.05-.563s-1.455.44-1.455 1.707c0 1.268.852 1.721 1.455 1.721s.96-.395 1.05-.57v.31c0 .103.084.186.188.186h.626a.187.187 0 00.188-.185v-2.916a.187.187 0 00-.188-.186zm-1.559 2.605c-.648 0-.731-.634-.731-.965 0-.326.082-.95.731-.95s.738.624.738.95c0 .33-.09.965-.738.965z" fill="url(#pi-SVGID_00000043436590437117869210000010855931133364857491_)"/><linearGradient id="pi-SVGID_00000175291399114651225350000013803905921597476495_" gradientUnits="userSpaceOnUse" x1="22.194" y1="16.8" x2="33.303" y2="11.923"><stop offset="0" stop-color="#00a035"/><stop offset="1" stop-color="#00d145"/></linearGradient><path d="M32.812 11.002h-.6a.188.188 0 00-.175.115l-.812 1.974-.786-1.972a.188.188 0 00-.175-.117h-.662a.186.186 0 00-.172.26l1.253 2.847-.516 1.123a.186.186 0 00.172.263h.628a.188.188 0 00.172-.11l1.845-4.122a.186.186 0 00-.172-.261z" fill="url(#pi-SVGID_00000175291399114651225350000013803905921597476495_)"/></svg>
```

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Looks good. I left some comments regarding optimizing the code.

**Review by tauthomas01 — APPROVED** [review]

**Comment by larouxn** [comment]
👋 Hello, your icon(s) have been released in the latest version of this library. The rollout process will begin within a few days. We wanted to have it/them live this week but looking more likely to be live early to mid next week. Will report back here when live. Thank you for your patience. 🙇‍♂️

**Comment by larouxn** [comment]
👋 Hello, actually your icon(s) should be live everywhere now. 🚀

**Pull request merged**